Gruntfile.js

```diff
@@ -74,10 +74,6 @@ module.exports = function (grunt) {
           // Source Files
           {expand: true, cwd: 'src/', src: ['**'], dest: JS + '/'},
 
-          // Courage files
-          {expand: true, cwd: 'packages/@okta/courage/src/', src: ['**'], dest: JS + '/shared/'},
-          {expand: true, cwd: 'packages/@okta/courage/src/vendor', src: ['**'], dest: JS + '/vendor/'},
```

Comment:
do we need to copy the new `courage-for-signin-widget.js` to `target/js` folder?

Reply:
hmm, either way works.
I thought it's unnecessary to copy but rather refer to the file directly.